tgui/packages/tgui/interfaces/Fabrication/DesignBrowser.tsx

```diff
@@ -492,6 +492,7 @@ const CategoryView = <T extends Design = Design>(
     <Section
       title={category.title}
       key={category.anchorKey}
+      container_id={category.anchorKey}
```

Comment:
It worked at some point.

Reply:
I guess it was before the react refactor then.  Not sure what changed

Reply:
I feel like the line directly above your new line, `key={category.anchorKey}`, is probably related to this.
Y'know, given it's almost the exact same.

Reply:
Looking through the code a bit it seems like it used to be this:
![image](https://github.com/tgstation/tgstation/assets/42909981/9b9750c2-158f-4fec-8a98-ae92d2390984)

Reply:
Found the exact pr: #80368

Reply:
Yup you are 100% correct. Because the `id` attribute was removed it was no longer set on the section `div`. I just re-added that functionality again

Reply:
Mhm, yeah, though I do wanna ask:
What *is* `key={category.anchorKey}` currently used for, then?
And, why not just use `id={category.anchorKey}` directly?

Reply:
2 things.

1) `id` was removed from Section entirely in that PR so you get compile time error now. Which is why i made my own variable to pass it into section directly

2) `key` is used by react internaly to differentiate between multiple components of the same type so it can render them correctly. In other words we need it for for rendering purposes only

Reply:
Huh, that's odd... I just tested it myself and it just worked.
Besides that I didn't see anything get removed from Section in that pr? Lemme check again.

Reply:
`id` definitly got removed

![Screenshot (399)](https://github.com/tgstation/tgstation/assets/110812394/9fe0c087-2429-431c-8e0b-68948d0063f7)

Anyway lets wait for maintainer feedback now

Reply:
wack, yeah fair enough then, good luck o7
I've been reading up on the related React bits during this convo, but personally I can't come up with an idea that's not more cluttered than this